examples/client/client.c

```diff
@@ -1445,6 +1444,49 @@ static void Usage(void)
 #endif
 }
 
+static int SimulateWantWriteIOSendCb(WOLFSSL *ssl, char *buf, int sz, void *ctx)
```

Comment:
Can you move this code in wolfssl/test.h with the other shared callbacks?  That way the server and client don't have to duplicate this code.

Reply:
Good idea.